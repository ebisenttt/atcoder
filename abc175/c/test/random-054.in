-287930642296563 975336176359693 791767424952610
