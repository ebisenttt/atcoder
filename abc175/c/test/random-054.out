503836782656047
